windows/src/main.cpp

```diff
@@ -124,127 +133,75 @@ int main() {
-				row_left = gray.at<uchar>(row_number, 159 + left);  
-				left--;
+			// diff_max was determined by finding the maxmimum diff that can be used to go from center to the very edge of the lane.
+			// It is an approximation. In testing, 65px was the farthest we could go from center in-game without losing lane.
+			int diff_max = 70;
 
-			}
-			if (row_right != 255) {
-				row_right = gray.at<uchar>(row_number, 159 + right); 
-				right++;
+			// jerk_factor = how fast the wheel will turn
+			// (1/70) = Limits steering to move 1px MAXMIMUM every time step (1 second).
+			double jerk_factor = 1 / 70;
 
-			}
-			i++;
+			// linearized_diff = diff on a scale of -1 to 1
+			double linearized_diff = diff / diff_max;
 
-		}
-		SetActiveWindow(hWnd);
+			// our new wheel position is determined by adding a number, turn_amount, to the previous wheel position
+			double turn_amount = linearized_diff * jerk_factor;
 
-		int average = (left == -150 || right == 150) ? 0 : left + right;
-		if (left + right < -50)
-		{
-			cout << "go left ";
-
-			INPUT input[2];
-			input[0].type = INPUT_KEYBOARD;
-			// Translating 'A' to Scan Code, then pressing down
-			input[0].ki.wScan = MapVirtualKey(0x41, MAPVK_VK_TO_VSC);
-			input[0].ki.dwFlags = KEYEVENTF_SCANCODE;
-			// Translating 'A' to Scan Code, then releasing key
-			input[1].ki.wScan = MapVirtualKey(0x41, MAPVK_VK_TO_VSC);
-			input[1].ki.dwFlags = KEYEVENTF_SCANCODE | KEYEVENTF_KEYUP;
-			SendInput(2, input, sizeof(INPUT));
-		}
-		else if (left + right > -50 && left + right < 50){
-			cout << "go straight ";
-			for (int x = 0, y = 0; x < 700 && y < 700; x += 10, y += 10)
-			{
-				/*
-				INPUT input[2]; // Using SendInput to send input commands
-				input[0].type = INPUT_KEYBOARD;
-				// Translating 'A' to Scan Code, then releasing key
-				input[0].ki.wScan = MapVirtualKey(0x41, MAPVK_VK_TO_VSC);
-				input[0].ki.dwFlags = KEYEVENTF_SCANCODE | KEYEVENTF_KEYUP;
-				// Translating 'D' to Scan Code, then releasing key
-				input[1].ki.wScan = MapVirtualKey(0x44, MAPVK_VK_TO_VSC);
-				input[1].ki.dwFlags = KEYEVENTF_SCANCODE | KEYEVENTF_KEYUP;
-				SendInput(2, input, sizeof(INPUT));
-				
+			if (turn_amount < .5){
+				turn_amount = 0;
 			}
-		}
-		/* else{
-			cout << "go right ";
-			{
-				INPUT input[2];
-				input[0].type = INPUT_KEYBOARD;
-				// Translating 'D' to Scan Code, then pressing down
-				input[0].ki.wScan = MapVirtualKey(0x44, MAPVK_VK_TO_VSC);
-				input[0].ki.dwFlags = KEYEVENTF_SCANCODE;
-				// Translating 'D' to Scan Code, then releasing key
-				input[1].ki.wScan = MapVirtualKey(0x44, MAPVK_VK_TO_VSC);
-				input[1].ki.dwFlags = KEYEVENTF_SCANCODE | KEYEVENTF_KEYUP;
-				SendInput(2, input, sizeof(INPUT));
+			else {
+				turn_amount = 1;
 			}
+			
+			int moveMouse = (pt.x + diffOld + turn_amount);
+
+			cout << "Steer: " << moveMouse << "px ";
+
+			SetCursorPos(moveMouse, height / 2);

```

Comment:
To simplify, we could create a variable to define "height / 2". For example
`int vertical_center = height / 2`